schema/Readme.md

```diff
@@ -1,67 +1,68 @@
+      - [Core Data Type to Connector](#core-data-type-to-connector)
+      - [Core Data Type to Generator](#core-data-type-to-generator)
+      - [List Types](#list-types)
+      - [Optional Types](#optional-types)
+      - [Relations](#relations)
+        - [One-to-One (1:1) Relationships](#one-to-one-11-relationships)
+        - [One-to-Many (1:N) Relationships](#one-to-many-1n-relationships)
+        - [Implicit Many-to-Many (M:N) Relationships](#implicit-many-to-many-mn-relationships)
+        - [Explicit Many-to-Many (M:N) Relationships](#explicit-many-to-many-mn-relationships)
+        - [Self-Referential Relationships](#self-referential-relationships)
+        - [Multiple-Reference Relationships](#multiple-reference-relationships)
+        - [Referencing Primary Composite Keys](#referencing-primary-composite-keys)
+        - [Referencing fields that are not @id](#referencing-fields-that-are-not-id)
+    - [Attributes](#attributes)
+      - [Case 1. No arguments](#case-1-no-arguments)
+      - [Case 2. One positional argument](#case-2-one-positional-argument)
+      - [Case 3. Many named arguments](#case-3-many-named-arguments)
+      - [Field Attributes](#field-attributes)
+      - [Core Field Attributes](#core-field-attributes)
+        - [@id](#id)
+        - [@unique](#unique)
+        - [@map(\_ name: String)](#map%5C_-name-string)
+        - [@default(\_ expr: Expr)](#default%5C_-expr-expr)
+        - [@relation(\_ name?: String, references?: Identifier[], onDelete?: CascadeEnum)](#relation%5C_-name-string-references-identifier-ondelete-cascadeenum)
+        - [@updatedAt](#updatedat)
+      - [Block Attributes](#block-attributes)
+      - [Core Block Attributes](#core-block-attributes)
+      - [Type Specifications](#type-specifications)
+    - [Why do we enforce the Core Prisma Primitive Type, even when there is a type specification?](#why-do-we-enforce-the-core-prisma-primitive-type-even-when-there-is-a-type-specification)
+  - [Comments](#comments)
+  - [Type Definition](#type-definition)
+  - [Enum Block](#enum-block)
+  - [Embed Block](#embed-block)
+    - [Inline Embeds](#inline-embeds)
+  - [Env Function](#env-function)
+    - [Introspect Behavior](#introspect-behavior)
+    - [Migrate Behavior](#migrate-behavior)
+    - [Generate Behavior](#generate-behavior)
+    - [Switching Datasources based on Environments](#switching-datasources-based-on-environments)
+  - [Function](#function)
+  - [Importing schemas](#importing-schemas)
+    - [Importing from other endpoints](#importing-from-other-endpoints)
+    - [Conflict Resolution](#conflict-resolution)
+  - [Auto Formatting](#auto-formatting)
+    - [Formatting Rules](#formatting-rules)
+      - [Configuration blocks are align by their `=` sign.](#configuration-blocks-are-align-by-their--sign)
+      - [Field definitions are aligned into columns separated by 2 or more spaces.](#field-definitions-are-aligned-into-columns-separated-by-2-or-more-spaces)
+- [FAQ](#faq)
```

Comment:
This points to the headline structure in that file being "strange", as `FAQ` seems to be second top level headline.